web/gui/dashboard_info.js

```diff
@@ -4908,6 +4908,77 @@ netdataDashboard.context = {
         info: 'The function <code>swap_writepage</code> is called when the kernel writes a page to swap memory. This chart is provided by eBPF plugin.'
     },
 
+    'services.file_open': {
+        info: 'Calls to the internal function <code>do_sys_open</code> ( For kernels newer than <code>5.5.19</code> we add a kprobe to <code>do_sys_openat2</code>. ), which is the common function called from' +
```

Comment:
```suggestion
        info: 'Calls to the internal function <code>do_sys_open</code> (for kernels newer than <code>5.5.19</code> we add a kprobe to <code>do_sys_openat2</code>), which is the common function called from' +
```

Reply:
Thank you, I fixed for `services` and `apps`.